mmv1/templates/terraform/examples/cloud_identity_groups_dynamic.tf.erb

```diff
@@ -0,0 +1,21 @@
+resource "google_cloud_identity_group" "<%= ctx[:primary_resource_id] %>" {
+  display_name         = "<%= ctx[:vars]['id_group'] %>"
+  initial_group_config = "WITH_INITIAL_OWNER"
+
+  parent = "customers/<%= ctx[:test_env_vars]['cust_id'] %>"
+
+  group_key {
+  	id = "<%= ctx[:vars]['id_group'] %>@<%= ctx[:test_env_vars]['org_domain'] %>"
+  }
+
+  labels = {
```

Comment:
I _suspect_ that when we run the test we'll see a diff on the dynamic label. If we don't I've missed something 🙂 

If that's the case, we'll want to add it here. We may also want to document explicitly in `dynamicGroupMetadata` something like "If this is set, X label with Y value **must** be set or Terraform will report a diff on the `labels` field."

Reply:
@bukzor-sentryio updated the docs 🙌